projects/admin/src/app/record/brief-view/loans-brief-view/loans-brief-view.component.html

```diff
@@ -53,14 +53,16 @@ <h5 class="ui:grow">{{ record.metadata.document.title | mainTitle }}</h5>
               @if (record.metadata?.is_late) {
                 <p-tag severity="danger" translate>overdue</p-tag>
                 ({{ 'due date' | translate }}
-                <span [ngClass]="{'text-error ui:font-bold': record.metadata?.is_late}">
-                  {{record.metadata.end_date | date: 'shortDate' }}</span>)
+                <!-- Double span to avoid gap before ')' -->
+                <span><span [ngClass]="{'text-error ui:font-bold': record.metadata?.is_late}">
+                  {{record.metadata.end_date | date: 'shortDate' }}</span>)</span>
```

Comment:
We'd have to find a more elegant solution in css, because it's weird to put 2 spans to fix this.

Reply:
Put all code on one line:
`({{ 'request expire at' | translate }} <span [ngClass]="{'text-error': isRequestExpired}">{{ record.metadata.request_expire_date | date: 'shortDate' }}</span>)`